src/pages/StocktakesPage.js

```diff
@@ -155,13 +159,16 @@ export const StocktakesPage = ({ routeName, currentUser, dispatch: reduxDispatch
       <View style={newPageTopSectionContainer}>
         <View style={newPageTopLeftSectionContainer}>
           <SearchBar
-            onChange={value => debouncedDispatch(filterData(value))}
+            onChangeText={onFilterData}
             style={searchBar}
             color={SUSSOL_ORANGE}
             placeholder=""
+            value={searchTerm}
```

Comment:
passing `style`, `color` and `placeholder` values which are equal to defaults

Reply:
darn. yes. missed that. it's in a few files isn't it?

Reply:
ya will just make an issue to fix em